### ISSUE TITLE
Implement an autocompiler on the server.

### ISSUE THREAD

**Issue by Chenterito** [Ezz-lol/boiii-free#18]
### Duplicate Issue Check

- [ ] I have searched the list of issues on this repository. I confirm that this is not a duplicate of any open or closed issue(s).

### Your feature request or suggestion:

Implement an autocompiler on the server. Since personally, I have been making some simple scripts, and since having to compile with a third tool, the script compiles, but when uploading to the server, it just closes and shows no error.
Additional, implement the ability to load custom scripts, not just replace existing scripts.

**Comment by diamante0018** [comment]
I agree, ezz boiii should make their own custom GSC compiler from zero.
But if your server is closing it's because you have bugs in your script. Try using the official mod tools maybe? lol

**Comment by Chenterito** [comment]
but how to correct, if I don't know what the error is.  that is the point.  when compiling the script, it has no errors.
